analytics.js

```diff
@@ -1351,6 +1384,22 @@ class Analytics {
 
     return returnObj;
   }
+
+  /**
+   * A public method to start a session
+   * @param {string} sessionId     session identifier
+   * @returns
+   */
+  startSession(sessionId) {
```

Comment:
Did we consider this?
// in this case we'll calculate the sessionId
rudderClient.startSession();

Reply:
Yes, in this function sessionId is optional.